### PR TITLE
add cli

### PR DESCRIPTION
**Pull request by xdssio**
* bump to 0.0.7 
* cli.md

### CODE REVIEW

**Comment by xdssio** [comment]
* Might wanna go over the tutorial, clean and add it.

**Comment by ylow** [comment]
A "getting started" will be great. Like "get an account [here]". Do this to get authed.
"xet repo make blah" to make a repo. 
Then "xet cp stuff in". 
Go to the website to see it.

**Review by ylow — APPROVED** [review]

File: docs/conf.py [hunk 1]
```diff
@@ -9,7 +9,7 @@
 project = 'pyxet'
 copyright = '2023, XetHub'
 author = '<a href="https://xethub.com/xdssio" target="_blank">Jonathan Alexander</a>, <a href="https://xethub.com/team" target="_blank">XetHub team</a>, and the <a href="https://github.com/xetdata/pyxet" target="_blank">pyxet</a> community'
-release = '0.0.6'
+release = '0.0.7'
```
Comment (rajatarya) [inline docs/conf.py:12]: I think this needs to be 0.0.8 now.

**Review by rajatarya — APPROVED** [review]
Are there docs for fork and duplicate?

Otherwise I think just version bump needed.

**Pull request merged**